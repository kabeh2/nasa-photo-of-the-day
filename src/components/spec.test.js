import React from "react";
import Info from "./info";
import { shallow, configure } from "enzyme";
import Adapter from "enzyme-adapter-react-16";
import renderer from "react-test-renderer";
import { act } from "react-dom/test-utils";

configure({ adapter: new Adapter() });

describe("Info Component", () => {
  it("matches the snapshot", () => {
    act(() => {
      const tree = renderer.create(<Info />).toJSON();

      expect(tree).toMatchSnapshot();
    });
  });
});
