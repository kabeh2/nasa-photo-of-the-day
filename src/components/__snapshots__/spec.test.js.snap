// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Info Component matches the snapshot 1`] = `
<div
  className="info-container"
>
  <div
    className="info-container--content"
  >
    <img
      alt="Nasa Logo"
      className="logo"
    />
    <div
      className="info"
    >
      <h1>
        NASA 
         of the Day
      </h1>
      <div
        className="info_title"
      />
      <div
        className="react-datepicker-wrapper"
      >
        <div
          className="react-datepicker__input-container"
        >
          <input
            className="datePicker"
            disabled={false}
            onBlur={[Function]}
            onChange={[Function]}
            onClick={[Function]}
            onFocus={[Function]}
            onKeyDown={[Function]}
            readOnly={false}
            type="text"
            value=""
          />
        </div>
      </div>
    </div>
  </div>
</div>
`;
